commit/plugin.go

```diff
@@ -349,7 +349,7 @@ func (p *Plugin) ShouldTransmitAcceptedReport(
 		return false, fmt.Errorf("validate merkle roots state: %w", err)
 	}
 
-	p.lggr.Debugw("transmitting report",
+	p.lggr.Infow("transmitting report",
```

Comment:
Making this one Info level seems fine.... not sure about the others. Making everything Info means there would be an avalanche of Info logs as well

Reply:
We'll have to revisit these later, for now I think this is slightly better than turning on Debug and getting all the debug logs of the app.